default.yml

```diff
@@ -41,5 +41,8 @@ Metrics/BlockLength:
 Metrics/LineLength:
   Max: 120
 
+RSpec/ImplicitSubject:
+  Enabled: false
```

Comment:
Is it possible to configure it instead of disabling?

Reply:
@AleksSenkou what do you mean about configuring? + all decisions have been discussed in the  issue

Reply:
@roman-dubrovsky my point here is that we just disable a rule, but not configure it.

Will this one:
```ruby
subject { service.call }

it 'test' do
  expect(subject).to eq value
end  
```
raise an error?

Reply:
and, it will but because `subject` does not have a name

Reply:
this is  a correct example 

```
subject(:tested_value) { service.call }

it 'test' do
  expect(tested_value).to eq value
end  
```

Reply:
Got it 👍